app/components/examplePage1/ExamplePage1Controller.js

```diff
@@ -1,12 +1,37 @@
 (function() {
   'use strict';
   // Controller naming conventions should start with an uppercase letter
-  function ExamplePage1Controller($scope) {
-    $scope.exampleContents1 = 'We are up and running using a required module!';
-    $scope.paragraphText = ' Example Page 1 explicitly references required files';
+  function ExamplePage1Controller($scope, $http) {
+    // Sum the non-itemized data over the summary pages, its in the row labled A,2.
+    var notItemized;
+    $http.get('https://data.oaklandnet.com/resource/rsxe-vvuw.json?$select=sum(amount_a)&$where=elect_date=\'2012-11-06T00:00:00\' and form_type=\'A\' and line_item=\'2\'').
+      success(function(sum) {
+        notItemized = sum[0]['sum_amount_a'];
+      });
+    // Sum by the entity code from schedule A
+    $http.get('https://data.oaklandnet.com/resource/3xq4-ermg.json?$select=sum(tran_amt1), entity_cd&$group=entity_cd&$where=elect_date=\'2012-11-06T00:00:00\'')
+      .success(function(data) {
+          // Calculate the grand total.
+          data = data.concat({ entity_cd: 'NIT', sum_tran_amt1: notItemized });
+          $scope.total = 0;
+          for (var i = 0; i < data.length; i++) {
+            $scope.total += Number(data[i]['sum_tran_amt1']);
+          }
+          data.sort(function(a, b) {
+            if (Number(a['sum_tran_amt1']) > Number(b['sum_tran_amt1'])) {
+              return -1;
+            }
+            if (Number(a['sum_tran_amt1']) < Number(b['sum_tran_amt1'])) {
+              return 1;
+            }
+            return 0;
+          });
```

Comment:
The best way to compare numbers is by using the difference. You could simplify this like

```
data.sort(function(a, b) {
  return  Number(b['sum_tran_amt1']) - Number(a['sum_tran_amt1']);
}
```